plugins/catalog-backend/src/processing/DefaultCatalogProcessingEngine.ts

```diff
@@ -215,7 +215,12 @@ export class DefaultCatalogProcessingEngine {
               const { entityRefs: parents } =
                 await this.processingDatabase.transaction(tx =>
                   this.processingDatabase.listParents(tx, {
-                    entityRef,
+                    entityRefs: [
+                      entityRef,
+                      ...result.deferredEntities.map(e =>
+                        stringifyEntityRef(e.entity),
```

Comment:
This makes me realize that the stableStringify function still is sensitive to array order; i wonder if in particular discovery locations are at risk of emitting entities in a stochastic order. If so, it might make sense to order this array and `result.deferredEntities` as well by ref 🤔 Being basically explicit in code about "the order of these do not constitute an effective change" so to speak. Could be done separately perhaps especially if we want to order `result.deferredEntities` too.

Reply:
Thinking separate change for that if we want to do it?

Reply:
Mm